org.kde.koko.json

```diff
@@ -0,0 +1,89 @@
+{
+    "id": "org.kde.koko",
+    "rename-icon": "koko",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15",
+    "sdk": "org.kde.Sdk",
+    "command": "koko",
+    "tags": ["nightly"],
+    "desktop-file-name-suffix": " (Nightly)",
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--filesystem=home",
+        "--env=XDG_DATA_DIRS=/usr/share:/app/share/"
+    ],
+    "separate-locales": false,
+
+    "modules": [
+        {
+            "name": "exiv2",
+            "buildsystem": "cmake-ninja",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://www.exiv2.org/builds/exiv2-0.27.2-Source.tar.gz",
+                    "sha256": "2652f56b912711327baff6dc0c90960818211cf7ab79bb5e1eb59320b78d153f"
+                }
+            ]
+        },
+        {
+            "name": "kio-extras",
+            "buildsystem": "cmake-ninja",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.kde.org/stable/release-service/21.04.1/src/kio-extras-21.04.1.tar.xz",
+                    "sha256": "4ebfca782ed5410903f6ec5bebb5d144447c4b3d50cb4e5f270354b136fe5c4c"
+                }
+            ]
+        },
+        {
+            "name": "kquickimageeditor",
+            "buildsystem": "cmake-ninja",
+            "sources": [
+                {
+                    "type": "archive",
+                    "sha256": "5b13f8e53bc7ea960b24064948f3d28bfb39f47529d5fa63a2090a047465c1ea",
+                    "url": "https://download.kde.org/stable/kquickimageeditor/0.1/kquickimageeditor-0.1.3.tar.xz"
+                }
+            ]
+        },
+        {
+            "name": "koko",
+            "buildsystem": "cmake-ninja",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.kde.org/stable/plasma-mobile/21.05/koko-21.05.tar.xz",
+                    "sha256": "65005ad5d2072df69bb2fd38d1de5b772cc7d132093a96f52dc1ed86e8fc1602"
+                },
+                {
+                    "type": "file",
+                    "path": "koko/cities1000.zip",
+                    "dest-filename": "src/cities1000.zip",
```

Comment:
what are these and why are they in the flatpak repository?

Reply:
Those are the data files for the geolocalisation features of Koko (group photos by localization they were taken). They are taken from http://download.geonames.org/ and added a bit of information about them in the README: https://invent.kde.org/graphics/koko#packaging

Reply:
As for the reason why they can't be downloaded as part of the flatpak recipe, geonames.org doesn't seem to provide any kind of snapshos, only ever changing data at the same url.

Reply:
That could use the flatpak data checker to auto-update the url maybe?

Reply:
As mentioned, the url doen't change, just the data behind it.